soundbay/train.py

```diff
@@ -103,7 +104,12 @@ def modeling(
         logger=logger
     )
 
+    # Freeze layers if required (num_unfrozen!=0)
+    if num_unfrozen:
```

Comment:
I think that the behaviour where `num_unfrozen == 0` means "everything can be trained" is confusing (I would expect it to mean "nothing can train", which makes no sense but is the natural way to interpert this parameter). I suggest that `num_unfrozen is None` would signify the "usual" case where nothing is frozen.